src/main/java/htsjdk/samtools/SAMRecordUtil.java

```diff
@@ -39,23 +39,38 @@
 
     /**
      * Reverse-complement bases and reverse quality scores along with known optional attributes that
-     * need the same treatment.  See {@link #TAGS_TO_REVERSE_COMPLEMENT} {@link #TAGS_TO_REVERSE}
+     * need the same treatment. This does not make a deep copy of the bases, qualities or attributes:
```

Comment:
Rather than talking about "deep copy", perhaps talk about "in-place" versus "makes a copy of".